### PR TITLE
Blinking btn

### PR DESCRIPTION
**Pull request by adrsh79086**
Issue:

### CODE REVIEW

**Review by dsnehasish74 — COMMENTED** [review]
Blink button should be above of bounce button . And for black and white button the animation is not working can you fix that ? Otherwise you did a great work . Thank you .

**Review by shahednasser — COMMENTED** [review]
In addition to @dsnehasish74 's comments, don't use the black text for all the buttons. If a button originally has a white text keep the white text, and if it has a black text then keep it black.

File: src/components/animated/_Blinking.less [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.blinking-btn&:hover {
```
Comment (shahednasser) [inline src/components/animated/_Blinking.less:1]: Remove the & here it's unnecessary

File: src/components/animated/_Blinking.less [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  animation: blinkingText 1.2s infinite;
+  border: none;
```
Comment (shahednasser) [inline src/components/animated/_Blinking.less:3]: remove this line

**Review by shahednasser — COMMENTED** [review]
Also please rename `src/components/animated/_Blinking.less` to `_blinking.less` without capitalizing the b.

**Pull request closed without merging**